x/schema/keeper/schema.go

```diff
@@ -40,7 +39,7 @@ func (k Keeper) LookUpContent(cid string, content interface{}) error {
 		return err
 	}
 
-	if err = json.Unmarshal(buf, &content); err != nil {
+	if err = content.Unmarshal(buf); err != nil {
```

Comment:
If `content` is nil, unmarshal may panic.

Reply:
good point